### PR TITLE
add proxy and proxy_auth varriables to ClientSession.__init__

### PR DESCRIPTION
**Pull request by meshya**
## What do these changes do?

Users are now able to set `proxy` and `proxy_auth` as defaults in `ClientSession.__init__`.

## Are there changes in behavior for the user?
No, there is not

These changes will not affect applications written with previous versions of aiohttp.

However, they can now set default proxy settings when creating an instance of `ClientSession`.

NOTICE: If default settings are set and then a new proxy setting is specified in` SessionClient.get` (or `post`, etc.), the latter will take effect.

## Is it a substantial burden for the maintainers to support this?

It is a simple patch and should not cause any issues.

## Checklist

- [X] I think the code is well written
- [X] Unit tests for the changes exist
- [X] Documentation reflects the changes
- [X] If you provide code modification, please add yourself to `CONTRIBUTORS.txt`
  * The format is &lt;Name&gt; &lt;Surname&gt;.
  * Please keep alphabetical order, the file is sorted by names.
- [x] Add a new news fragment into the `CHANGES/` folder
  * name it `<issue_or_pr_num>.<type>.rst` (e.g. `588.bugfix.rst`)
  * if you don't have an issue number, change it to the pull request
    number after creating the PR
    * `.bugfix`: A bug fix for something the maintainers deemed an
      improper undesired behavior that got corrected to match
      pre-agreed expectations.
    * `.feature`: A new behavior, public APIs. That sort of stuff.
    * `.deprecation`: A declaration of future API removals and breaking
      changes in behavior.
    * `.breaking`: When something public is removed in a breaking way.
      Could be deprecated in an earlier release.
    * `.doc`: Notable updates to the documentation structure or build
      process.
    * `.packaging`: Notes for downstreams about unobvious side effects
      and tooling. Changes in the test invocation considerations and
      runtime assumptions.
    * `.contrib`: Stuff that affects the contributor experience. e.g.
      Running tests, building the docs, setting up the development
      environment.
    * `.misc`: Changes that are hard to assign to any of the above
      categories.
  * Make sure to use full sentences with correct case and punctuation,
    for example:
    ```rst
    Fixed issue with non-ascii contents in doctest text files
    -- by :user:`contributor-gh-handle`.
    ```

    Use the past tense or the present tense a non-imperative mood,
    referring to what's changed compared to the last released version
    of this project.

### CODE REVIEW

**Comment by Dreamsorcerer** [comment]
Seems reasonable, but please resolve coverage.

**Comment by meshya** [comment]
Ok, I try fix it today

File: CHANGES/9207.feature.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Add default proxy variables to ClientSession -- by :user:`meshya`.
```
Comment (Dreamsorcerer) [inline CHANGES/9207.feature.rst]:
```suggestion
Added ``proxy`` and ``proxy_auth`` parameters to ``ClientSession`` -- by :user:`meshya`.
```

**Comment by Dreamsorcerer** [comment]
Thanks, if you could do the backport as above, that'd be a great help.

**Comment by meshya** [comment]
Thanks for merge,  I appreciate your time and effort, I'll do the backport soon.

**Pull request merged**